operator/internal/controller/loki/alertingrule_controller.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/grafana/loki/operator/internal/controller/loki/internal/lokistack"
 )
 
+const ControllerNameAlertingRule = "alertingrule"
```

Comment:
@xperimental I've kept the names that were being used in the loggers. However if we will generate metrics labels out of these what do you think of making them more easily searchable with a `lokistack` or `loki-operator` prefix. Let me know WDYT
```suggestion
const ControllerNameAlertingRule = "alertingrule"
```

Reply:
This does not sound bad, but I think I'd make this a separate change if it is not necessary for this PR to function.

Reply:
The renaming part is not necessary, the having to name all the controllers is. I'm fine with doing the renaming on a follow up